src/vaadin-number-field.html

```diff
@@ -325,29 +339,42 @@
           }
         }
 
+        /** @private */
         _allowed(sign) {
           const incr = sign * (this.step || 1);
           const value = parseFloat(this.value);
           return !this.value || (!this.disabled && this._incrementIsInsideTheLimits(incr, value));
         }
 
-        _stepChanged(step) {
+        /**
+         * @param {!number} newVal
+         * @param {!number} oldVal
+         * @protected
+         */
+        _stepChanged(newVal, oldVal) {
           // Avoid using initial value in validation
           this.__validateByStep = this.__stepChangedCalled || this.getAttribute('step') !== null;
-          this.inputElement.step = this.__validateByStep ? step : 'any';
+          this.inputElement.step = this.__validateByStep ? newVal : 'any';
 
           this.__stepChangedCalled = true;
-          this.setAttribute('step', step);
+          this.setAttribute('step', newVal);
         }
 
+        /** @private */
         _minChanged(min) {
           this.inputElement.min = min;
         }
 
+        /** @private */
         _maxChanged(max) {
```

Comment:
I'm not sure why these are private when other property observers like `_stepChanged` and `_valueChanged` are protected?

Reply:
Those observers that are overridden by `vaadin-integer-field` are marked as protected.

I would prefer to keep others as private unless we need to override them as well.